tests/utils/profilehandler/interface.go

```diff
@@ -0,0 +1,48 @@
+	Tagging                       bool   `yaml:"tagging,omitempty" json:"tagging,omitempty"`
+	Labeling                      bool   `yaml:"labeling,omitempty" json:"labeling,omitempty"`
+	Etcd                          bool   `yaml:"etcd,omitempty" json:"etcd,omitempty"`
+	FIPS                          bool   `yaml:"fips,omitempty" json:"fips,omitempty"`
+	STS                           bool   `yaml:"sts,omitempty" json:"sts,omitempty"`
+	Autoscale                     bool   `yaml:"autoscale,omitempty" json:"autoscale,omitempty"`
+	MultiAZ                       bool   `yaml:"multi_az,omitempty" json:"multi_az,omitempty"`
+	BYOVPC                        bool   `yaml:"byovpc,omitempty" json:"byovpc,omitempty"`
+	PrivateLink                   bool   `yaml:"private_link,omitempty" json:"private_link,omitempty"`
+	Private                       bool   `yaml:"private,omitempty" json:"private,omitempty"`
+	BYOK                          bool   `yaml:"byok,omitempty" json:"byok,omitempty"`
+	ETCDKMS                       bool   `yaml:"etcd_kms,omitempty" json:"etcd_kms,omitempty"`
+	NetWorkingSet                 bool   `yaml:"networking_set,omitempty" json:"networking_set,omitempty"`
+	Proxy                         bool   `yaml:"proxy,omitempty" json:"proxy,omitempty"`
+	HCP                           bool   `yaml:"hcp,omitempty" json:"hypershift,omitempty"`
+	OIDCConfig                    string `yaml:"oidc_config,omitempty" json:"oidc_config,omitempty"`
+	ProvisionShard                string `yaml:"provision_shard,omitempty" json:"provision_shard,omitempty"`
+	Ec2MetadataHttpTokens         string `yaml:"imdsv2,omitempty" json:"imdsv2,omitempty"`
+	AuditLogForward               bool   `yaml:"auditlog_forward,omitempty" json:"auditlog_forward,omitempty"`
+	AdminEnabled                  bool   `yaml:"admin_enabled,omitempty" json:"admin_enabled,omitempty"`
+	ExpirationTime                int    `yaml:"expiration_time,omitempty" json:"expiration_time,omitempty"`
+	VolumeSize                    int    `yaml:"volume_size,omitempty" json:"volume_size,omitempty"`
+	BillingAccount                string `yaml:"billing_account,omitempty" json:"billing_account,omitempty"`
+	ClusterAutoscalerEnabled      bool   `yaml:"cluster_autoscaler_enabled,omitempty" json:"cluster_autoscaler_enabled,omitempty"`
+	DisableUserWorKloadMonitoring bool   `yaml:"disable_uwm,omitempty" json:"disable_uwm,omitempty"`
+	SharedVPC                     bool   `yaml:"shared_vpc,omitempty" json:"shared_vpc,omitempty"`
+	DisableSCPChecks              bool   `yaml:"disable_scp_checks,omitempty" json:"disable_scp_checks,omitempty"`
+	AdditionalSGNumber            int    `yaml:"additional_sg_number" json:"additional_sg_number,omitempty"`
+	WorkerPoolReplicas            int    `yaml:"replicas" json:"replicas,omitempty"`
+	ExternalAuthConfig            bool   `yaml:"external_auth_config" json:"external_auth_config,omitempty"`
```

Comment:
Please order the attributes by type, and by name.

Reply:
Do you have any tool suggested to format?

Reply:
Updated